core/zms/src/main/rdl/Policy.tdl

```diff
@@ -16,6 +36,7 @@ type Assertion Struct {
     AssertionEffect effect (optional, default=ALLOW); //the effect of the assertion in the policy language
     Int64 id (optional); //assertion id - auto generated by server. Not required during put operations.
     Bool caseSensitive (optional); // If true, we should store action and resource in their original case
+    AssertionConditions assertionConditions (optional); // optional list of assertion conditions associated with given assertion
```

Comment:
we should name this just conditions otherwise we're repeating assertion part and be consistent with other fields (e.g. we're calling effect and not assertionEffect).